components/hab/src/command/pkg/export/helm.rs

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2017 Chef Software Inc. and/or applicable contributors
```

Comment:
2018 here and in all the new files?

Reply:
If it states just the year 2017, the update is not needed from what I understand.

Reply:
It was rather about whether the code was written in 2018, but 2017 was used out of habit.

Reply:
no code was written in 2018. :)